addons/editor-square-inputs/userscript.js

```diff
@@ -0,0 +1,48 @@
+export default async function ({ addon, msg, global, console }) {
```

Comment:
None of the imports are used except `addon`, but I would keep `console`.
```suggestion
export default async function ({ addon, console }) {
```